### ISSUE TITLE
[FEATURE REQUEST] Docker image ⚓

### ISSUE THREAD

**Issue by Meldiron** [bsnk-dev/boltz-engine#3]
Hello there 👋

The easiest way to start using pretty much anything server-related is if we have a simple `docker-compose up` script to start everything up in 1 command. Have you considered adding `docker.compose.yml`, `Dockerfile` and building Docker images with every release?

**Comment by Meldiron** [comment]
I just tried to implement it at https://github.com/Meldiron/boltz-engine/tree/feat-docker but can't get the build to work. If I am not mistaken, there is `public` folder with a web interface, but it is in `.gitignore` so we don't have this folder on Github. Can you confirm that this is the case? Can the public folder be pushed to Github?

Currently, response for `http://localhost:5000/` is `Not Found` and for `http://localhost:8000/` it is `Invalid URL!` with these logs in console:
```
Listening for admin actions on port 5000, http://localhost:5000
(38) Listening for execution requests on port 8000

Error: ENOENT: no such file or directory, stat '/app/public/index.html'
(21) [api/execute/favicon.ico] Failed to get instance of called function favicon.ico, instance not found
Error: ENOENT: no such file or directory, stat '/app/public/index.html'
```

**Comment by bsnk-dev** [comment]
@Meldiron I decided it makes more sense to push it to GitHub. It's fixed in 0aed155.
